### ISSUE TITLE
Pricechecking fails at times

### ISSUE THREAD

**Issue by LordExelot** [pabletos/Hubot-Warframe#41]
I am able to give you a lot of detail on this but I've seen it occur multiple times, when the /pc uses more than 1 word like /pc Arcane Grace it will fail, but also things like /pc flight(or toxic) for toxic flight will fail to find the item I am looking for aswell

**Comment by TobiTenno** [comment]
yeah, I said to submit it to the genesis form, since it isn't handled on here. I explained before that it's not in the mods that are looked up and to ask me to ask the person who runs the website that this runs off of to add it. it's based off of https://nexus-stats.com

**Comment by TobiTenno** [comment]
fixed on new bot, please close.
